rasa_nlu/converters.py

```diff
@@ -153,12 +154,24 @@ def rasa_nlu_data_schema():
         "required": ["text"]
     }
 
+    regex_feature_schema = {
```

Comment:
will have a look. need to implement the reading in of luis regular expressions from their format anyway.